stdlib/lang-parser/README.md

```diff
@@ -0,0 +1,309 @@
+## Ballerina-lang-parser
+
+The parser is implemented following the LL(K) recursive Descent parsing pattern. This pattern analyses the syntactic structure using k>1 lookahead tokens.
+
+**Token Structure**
+```ballerina
+Token record {
+    int tokenType;
+    string text;
+    int startPos;
+    int endPos;
+    int lineNumber;
+    int index;
+    string? whiteSpace;
+}
+```
+
+Grammar supported by the current implementation is given below.
```

Comment:
Let's move this to after the heading `**Parser Grammar**` and say 
```suggestion
The current implementation supports the following grammar.
```

Reply:
Ballerina lang-parser implementation in no longer maintained and has already being removed from the ballerina-lang repo.